maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -415,8 +417,11 @@ class IOSDriver(
 
     override fun inputText(text: String) {
         metrics.measured("operation", mapOf("command" to "inputText")) {
-            // silently fail if no XCUIElement has focus
-            runDeviceCall("inputText") { iosDevice.input(text = text) }
+            try {
+                runDeviceCall("inputText") { iosDevice.input(text = text) }
+            } catch (badRequest: XCUITestServerError.BadRequest) {
+                LOGGER.info("Bad request for input text, reason: ${badRequest.message}")
```

Comment:
why logger.info instead of logger.error?

Reply:
Fixed